administrator/language/de-DE/com_scheduler.ini

```diff
@@ -85,8 +85,13 @@ COM_SCHEDULER_MANAGER_TASKS="Geplante Aufgaben"
 COM_SCHEDULER_MANAGER_TASK_EDIT="Aufgabe bearbeiten"
 COM_SCHEDULER_MANAGER_TASK_NEW="Neue Aufgabe"
 COM_SCHEDULER_MANAGER_TOOLTIP_TASK_FAILING="Aufgabe fehlgeschlagen. Exit-Code: %1$d"
+COM_SCHEDULER_MSG_DUETASKS="Es gibt mindestens eine fällige Aufgabe, die bereits ausgeführt worden sein sollte. Bitte sicherstellen, dass mindestens ein Cron-Scheduler aktiviert ist und läuft."
```

Comment:
```suggestion
COM_SCHEDULER_MSG_DUETASKS="Es gibt mindestens eine Aufgabe, die bereits ausgeführt worden sein sollte. Bitte sicherstellen, dass mindestens ein Cron-Scheduler in den Einstellungen der Komponente aktiviert ist und läuft."
```

Reply:
Den Linux-Cron kann man ja nicht in der Komponente aktivieren. Daher war das etwas offener geschrieben.

Reply:
Stimmt dann passt es so 👍